website/docs/community/community-docs/maintaining-a-channel.md

```diff
@@ -11,11 +11,11 @@ There are three things you should do to be a good channel maintainer in the [dbt
 - Keep an eye out in #introductions for folks who might benefit from your new channel. For example if someone works in the space or on  the same problems, then direct them to the channel.
 - Make sure folks follow the [Rules of the Road](https://docs.getdbt.com/docs/contributing/slack-rules-of-the-road) in the channel. If you notice someone is not following one, gently remind them of the rule in thread, and, ideally, provide an example of how they can rephrase their message or where they can redirect it.  If you have a question about how to proceed, just post it in #ask-a-moderator with a screenshot or link to the thread and someone will give you advice.
 
-## Scope of the Role
+## Scope of the role
 
 A maintainer can be a dbt Labs employee, but does not have to be. *Slack channel maintainer* is philosophically similar to OSS maintainer. At the onset, the channel maintainer will help build up this new space in Slack and stir up conversation during the first few weeks of the channel's existence. They are someone who stays on top of feedback and encourages generative contributions. This is not necessarily someone who is the generator of content and contributions, or answers every question. 
 
-## Initial Instructions
+## Initial instructions
 
 1. Review the [Rules of the Road](https://docs.getdbt.com/docs/contributing/slack-rules-of-the-road) and [Code of Conduct](https://www.getdbt.com/community/code-of-conduct) and please let the the folks who created the channel know that you read both documents and you agree to be mindful of them. 
 2. If you are a vendor, review the [Vendor Guidelines](https://www.getdbt.com/community/vendor-guidelines).
```

Comment:
@gwenwindflower I can't see it deleted in the diff, but https://deploy-preview-2069--docs-getdbt-com.netlify.app/community/vendor-guidelines doesn't exist anymore. Was this you?

Reply:
wasn't me, that appears to be a WWW resource, might want to check with Mermell